BookingsApi/BookingsApi.AcceptanceTests/Models/AddParticipantRequest.cs

```diff
@@ -12,6 +12,7 @@ public static AddParticipantsToHearingRequest BuildRequest()
             var participants = Builder<ParticipantRequest>.CreateListOfSize(1).All()
                 .With(x => x.ContactEmail = $"Automation_{Faker.RandomNumber.Next()}@hmcts.net")
                 .With(x => x.Username = $"Automation_{Faker.RandomNumber.Next()}@hmcts.net")
+                .With(x => x.TelephoneNumber = "01234567890")
```

Comment:
Should this field also be called `ContactTelephone` like in Justice User?

Reply:
This pre-dates the justice user. Justice user should have probably been consistent with participant but they are entirely different entities so it probably use the language specific to the context. Hope that makes sense